libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/camera/DynamicCamera.java

```diff
@@ -22,8 +22,8 @@
 
   private static final double MAX_CAMERA_TILT = 60d;
   private static final double MIN_CAMERA_TILT = 45d;
-  private static final double MAX_CAMERA_ZOOM = 16d;
-  private static final double MIN_CAMERA_ZOOM = 12d;
+  private static final double MAX_CAMERA_ZOOM = 18d;
+  private static final double MIN_CAMERA_ZOOM = 18d;
```

Comment:
This change looks off to me - MIN and MAX zoom levels can't be equal to each other, otherwise, the `DynamicCamera` will stay at a constant zoom level.

Reply:
increased _Min_ camera zoom to `16`, and _Max_ to `20` cc @kmadsen